### PR TITLE
Fixed 2gp to call installer scripts

### PR DESCRIPTION
**Pull request by MatthewBlanski**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes
Run qa_org_2gp against a prerelease org shape.

### CODE REVIEW

**Review by m-kinkade — APPROVED** [review]

File: scripts/configure_k12_kit.cls [hunk 1]
```diff
@@ -1,6 +1,33 @@
...
+    public Boolean isPush(){
+        return true;
+    }
+    public Version previousVersion(){
+        return null;
+    }
+}
+
+//Runs install apex scripts for any context
 public static void executeInstallApex() {
```
Comment (xinnanlisf) [inline scripts/configure_k12_kit.cls:20]: Minor minor one. We might want to name this method more specifically as executeK12InstallApex. executeInstallApex sounds like a wrapper to me. However, ignore it if this is a temporary fix.

File: cumulusci.yml [hunk 2]
```diff
@@ -449,16 +466,16 @@ flows:
     config_unmanaged:
         steps:
             1:
-                task: enable_course_connections
```
Comment (xinnanlisf) [inline cumulusci.yml:452]: Really curious about the reason why we are reordering these tasks.

File: cumulusci.yml [hunk 3]
```diff
@@ -504,6 +521,17 @@ flows:
...
                 task: deploy_trial_config
 
+    qa_org_2gp:
+        steps:
+            2:
+                flow: config_2gp
+            3:
+                task: None
+            4:
+                flow: config_qa
```
Comment (xinnanlisf) [inline cumulusci.yml:531]: I think what Samsheeth points out in EDA repo also applies here.

File: cumulusci.yml (hunk 2 above)
Comment (MatthewBlanski) [reply cumulusci.yml:452]: This was deliberate as the install script should run before any other scripts to address dependencies or potential dependencies that we expose.

**Comment by xinnanlisf** [comment]
DO NOT merge until further notice.

**Comment by MatthewBlanski** [comment]
> DO NOT merge until further notice.

We should be able to merge this now.

**Comment by xinnanlisf** [comment]
@MatthewBlanski looks like the merge conflicts still persist

**Comment by MatthewBlanski** [comment]
This is a false positive. Not sure why it's identified...
Matthew Blanski
Lead Member of Technical Staff
Salesforce - Salesforce.org
C: 773-326-7278

**Pull request closed without merging**